public/javascript/posts/BookAppointment.js

```diff
@@ -19,6 +19,24 @@ var app = angular.module('BookAppointmentApp');
     ];
 
     $scope.viewRenderWeekAgenda = function(view, element){
+      if(view.type == 'month'){

```

Comment:
Please handle this in `dayEvent` (preferable) or `alertOnEventClick` and not in this function